jest bug with 1.1.10

Starting with the 1.1.10, jest produce this log : 

```
Cannot find module './PromiseStatuses' from 'promiseState.js'

    However, Jest was able to find:
    	'./promiseState.js'

    You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['ts', 'tsx', 'js'].

    See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string
```
This bug seems to appear on linux, but not on windows (windows is not case sensitivite) ...

Any idea ?

For the moment, I use 1.1.9 .

--- a/lib/index.js
+++ b/lib/index.js
@@ -5,7 +5,7 @@ const {
     PROMISE_PENDING,
     PROMISE_RESOLVED,
     PROMISE_REJECTED
-} = require('./PromiseStatuses');
+} = require('./promiseStatuses');
 
 const {
     PromisePredicates,

--- a/lib/promisePredicates/isPromiseNotPending.js
+++ b/lib/promisePredicates/isPromiseNotPending.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_PENDING} = require('../PromiseStatuses');
+const {PROMISE_PENDING} = require('../promiseStatuses');
 
 async function isPromiseNotPending(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promisePredicates/isPromiseNotRejected.js
+++ b/lib/promisePredicates/isPromiseNotRejected.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_REJECTED} = require('../PromiseStatuses');
+const {PROMISE_REJECTED} = require('../promiseStatuses');
 
 async function isPromiseNotRejected(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promisePredicates/isPromiseNotResolved.js
+++ b/lib/promisePredicates/isPromiseNotResolved.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_RESOLVED} = require('../PromiseStatuses');
+const {PROMISE_RESOLVED} = require('../promiseStatuses');
 
 async function isPromiseNotResolved(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promisePredicates/isPromisePending.js
+++ b/lib/promisePredicates/isPromisePending.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_PENDING} = require('../PromiseStatuses');
+const {PROMISE_PENDING} = require('../promiseStatuses');
 
 async function isPromisePending(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promisePredicates/isPromiseRejected.js
+++ b/lib/promisePredicates/isPromiseRejected.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_REJECTED} = require('../PromiseStatuses');
+const {PROMISE_REJECTED} = require('../promiseStatuses');
 
 async function isPromiseRejected(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promisePredicates/isPromiseResolved.js
+++ b/lib/promisePredicates/isPromiseResolved.js
@@ -1,5 +1,5 @@
 const {promiseStatus} = require('../promiseStatus');
-const {PROMISE_RESOLVED} = require('../PromiseStatuses');
+const {PROMISE_RESOLVED} = require('../promiseStatuses');
 
 async function isPromiseResolved(promise) {
     const status = await promiseStatus(promise);

--- a/lib/promiseState.js
+++ b/lib/promiseState.js
@@ -2,7 +2,7 @@ const {
     PROMISE_PENDING,
     PROMISE_RESOLVED,
     PROMISE_REJECTED
-} = require('./PromiseStatuses');
+} = require('./promiseStatuses');
 
 const pendingState = {
     status: PROMISE_PENDING

--- a/test/lib/promiseState.test.js
+++ b/test/lib/promiseState.test.js
@@ -3,7 +3,7 @@ const {
     PROMISE_PENDING,
     PROMISE_RESOLVED,
     PROMISE_REJECTED
-} = require('../../lib/PromiseStatuses');
+} = require('../../lib/promiseStatuses');
 
 describe('lib/promiseState', () => {
     it('should be a function', () => {

--- a/test/lib/promiseStatus.test.js
+++ b/test/lib/promiseStatus.test.js
@@ -3,7 +3,7 @@ const {
     PROMISE_PENDING,
     PROMISE_RESOLVED,
     PROMISE_REJECTED
-} = require('../../lib/PromiseStatuses');
+} = require('../../lib/promiseStatuses');
 
 describe('lib/promiseStatus', () => {
     it('should be a function', () => {

